locksmith/__tests__/app.test.ts

```diff
@@ -1,7 +1,7 @@
-const request = require('supertest')
-const app = require('../src/app')
-const Lock = require('../src/lock')
-const Transaction = require('../src/transaction')
+import request = require('supertest')
+import app from '../src/app'
```

Comment:
For modules we create internally, there is no reason not to make them be ES6, so I converted a few of them to see how it would go. More on that later.